### PR TITLE
Test brew fltk

### PR DESCRIPTION
**Pull request by KaruroChori**
Just for CI

### CODE REVIEW

**Comment by KaruroChori** [comment]
Somehow, while installing and detecting fltk on the system, it fall back to the custom compiled version, and even worse it forces X11. This might actually be related to #42.

**Comment by andy5995** [comment]
Would using '--wrap-mode nofallback' help? https://github.com/theimpossibleastronaut/rmw/blob/cc3f7bc1cd4dcb427ef3dcea90a7de7454974eb1/.github/workflows/c-cpp.yml#L141C26-L141C48

**Comment by andy5995** [comment]
> Somehow, while installing and detecting fltk on the system, it fall back to the custom compiled version, and even worse it forces X11. This might actually be related to #42.

I don't see anywhere in the meson.build where it looks for an installed fltk. I think you need something like this:

```
sqlitecpp_dep = dependency('sqlitecpp', required: false)
if not sqlitecpp_dep.found()
    sqlitecpp_dep = cxx.find_library('SQLiteCpp', required: false)
    if not sqlitecpp_dep.found()
        sqlitecpp_dep = dependency(
            'sqlitecpp',
            fallback: ['sqlitecpp', 'sqlitecpp_dep'],
            default_options: 'default_library=static',
        )
    endif
endif

sqlite_dep = dependency(
    'sqlite3',
    version: '>=3.0.0',
    fallback: ['sqlite3', 'sqlite3_dep'],
    default_options: 'default_library=static',
)
```

File: meson.build [hunk 1]
```diff
@@ -29,14 +61,14 @@ if need_pugixml or get_option('use_system_pugixml') == false
 endif
 
 if get_option('use_system_fltk')
-    libfltk_dep = dependency('fltk', version : '>=1.4', required: false)
-    libfltk_images_dep = dependency('fltk-images', version : '>=1.4', required: false)
-    libfltk_forms_dep = dependency('fltk-images', version : '>=1.4', required: false)
-    if libfltk_dep.found()==false or libfltk_images_dep.found()==false or libfltk_forms_dep.found()==false 
-        need_fltk=true
+    libfltk_dep = dependency('fltk', version: '>=1.4', required: false)
```
[the remark this answers is not in the document]
Comment (KaruroChori) [reply meson.build:64]: Yes, in theory I implemented some logic to search for the system default with a compatible version (unless forced not to). Else it uses the one provided via wrap.

All this in theory.

**Pull request closed without merging**